translations/en.json

```diff
@@ -1768,7 +1780,7 @@
   "network-preferences": "Network preferences",
   "network-preferences-desc-1": "Select which networks this address is happy to receive funds on",
   "network-preferences-desc-2": "Select which networks to receive funds on",
-  "network-settings": "Network settings",
+  "network-settings": "Networks",
```

Comment:
Suggestion: It's best we create a separate key for `networks` as if any usage arises for `network-settings` in future, we don't have to play around with names.

Reply:
Just checked and we already have a `networks` key for translations, but the intention here is for the keys to refer to the context of the translation instead of the translation itself. This way if in the future the text changes from "Networks" to idk "Network configuration"/"Network settings"/etc, we only have to change the text, not the key and all its usages. IMO translation keys should rarely change, otherwise it'll be a nightmare for the translators (unless they're AI of course).